packages/dds/tree/src/simple-tree/treeNodeApi.ts

```diff
@@ -101,11 +102,19 @@ export interface TreeNodeApi {
 	readonly status: (node: TreeNode) => TreeStatus;
 
 	/**
-	 * Returns the {@link FieldKind.Identifier | identifier} of the given node in the most compressed form possible.
+	 * Returns the {@link SchemaFactory.identifier | identifier} of the given node in the most compressed form possible.
 	 * @remarks
-	 * If the node's identifier is a valid `StableNodeKey`, then this will return a unique process-local integer corresponding to that identifier.
-	 * If the node's identifier is any other string, then this will return that string.
-	 * If the node has no identifier (that is, it has no field of an {@link FieldKind.Identifier | identifier} field kind), then this returns undefined.
+	 * If the node's identifier is a valid UUID that was automatically generated by the SharedTree, then this will return a unique process-local integer corresponding to that identifier.
+	 * This is useful for performance-sensitive scenarios involving many nodes with identifiers that need to be compactly retained in memory or used for efficient lookup.
+	 *
+	 * If the node's identifier is any other user-provided string, then this will return that string.
+	 *
+	 * If the node has no identifier (that is, it has no {@link SchemaFactory.identifier | identifier} field), then this returns `undefined`.
+	 *
+	 * If the node has more than one identifier, then this will throw an error.
```

Comment:
When can this happen? Might be worth elaborating on how developers can avoid this scenario.

Reply:
Good question. I don't really know why a developer would put multiple ID fields on a node. But I just wanted to make sure they know what happens if they do. We could disallow two ID fields from the get-go (error when building the schema), but I didn't want to be more restrictive than necessary in case somebody did have a weird reason to do it.